### PR TITLE
#4 Mass transfers split into chunks

### LINKED ISSUE #4: Mass transfers split into chunks
### Problem
Distributions that require more than 100 transactions will fail

### Solution
Split the transactions into smaller chunks and broadcast them individually

### PR DESCRIPTION
**Pull request by bravetarget**
### Changes

Resolves #4 

- Quick fix to work for distributions that require more than 100 transactions

### Checklist

- [ ] UAT

### CODE REVIEW

File: __tests__/waves.test.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Waves', () => {
   it('should apply the multiplier to distribution amounts', async () => {
     const massTx = Waves.massTransferDistribution(mockDistribution, 'test seed');
 
-    expect(massTx).toMatchObject({
+    expect(massTx).toMatchObject([{
```
Comment (michalmikolajczyk) [inline __tests__/waves.test.js]: good

File: src/lib/utility.js [hunk 2]
```diff
@@ -6,4 +6,16 @@ export default {
       }, ms);
     });
   },
+
+  chunkArray (arr, chunkSize) {
```
Comment (michalmikolajczyk) [inline src/lib/utility.js]: this is pretty cool

File: src/lib/waves.js [hunk 3]
```diff
@@ -55,10 +57,13 @@ const Waves = {
       recipient,
       amount: distribution[recipient] * (DISTR_MULTIPLIER || 1),
     }));
-    return massTransfer({
-      transfers,
-      assetId: TARGET_ASSET_ID,
-    }, seed);
+
+    return utility.chunkArray(transfers, 100).map(chunk => {
+      return massTransfer({
```
Comment (michalmikolajczyk) [inline src/lib/waves.js]: you could probably just return the `massTransfer({…` call on line 61

**Comment by michalmikolajczyk** [comment]
how can we UAT?

**Comment by bravetarget** [comment]
@michalmikolajczyk this tool was intended to be ran locally via cli, prob the best way to do a UAT is to check this branch out and run it.  ty for reviewing!!

**Pull request merged**